### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by MikaelSmith**
Pulls in the latest dependencies from puppetlabs/clj-parent.

### CODE REVIEW

**Comment by MikaelSmith** [comment]
Are there some plans to add Snyk to this repo?

**Comment by Magisus** [comment]
Yes eventually. We've been doing it piecemeal, and I think there might be a workflow update incoming to allow testing PR contents on PRs from forks, so I've been holding off a bit on doing more, but I can add it here now if that would be helpful.

**Comment by Magisus** [comment]
Looks like this also needs the java security file updates. I can add those too.

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,3 @@ env:
 script: lein $ADDITIONAL_ARGS test :all
 notifications:
   email: false
-  hipchat:
```
Comment (MikaelSmith) [inline .travis.yml:12]: 😮

**Comment by Magisus** [comment]
I've requested to get the Snyk tokens added. Let's see if that last commit fixes the FIPS issues.

Update, looks good.

**Review by mwaggett — APPROVED** [review]

**Comment by Magisus** [comment]
Jeremy will add the Snyk secrets tomorrow, if we're okay waiting on this. Otherwise I can pull out that commit and do it separately?

**Comment by MikaelSmith** [comment]
No rush on my end.

**Comment by Magisus** [comment]
Snyk is green, going to update to `pull_request_target`.

**Review by mwaggett — APPROVED** [review]

**Pull request merged**